### ISSUE TITLE
Fails to build after #1327, must be fixed before releasing v0.9.10

### ISSUE THREAD

**Issue by metalefty** [neutrinolabs/xrdp#1335]
It is easy to fix but a showstopper. Postpone release until this is fixed. I'm working on this, just for the record.
```
Making all in chansrv
  CC       chansrv.o
  CC       chansrv_common.o
  CC       chansrv_fuse.o
  CC       clipboard.o
  CC       clipboard_file.o
  CC       devredir.o
  CC       fifo.o
  CC       irp.o
In file included from irp.c:30:
In file included from ./irp.h:27:
./chansrv_fuse.h:39:5: error: unknown type name 'time_t'
    time_t          atime;             /* Time of last access.              */
    ^
./chansrv_fuse.h:40:5: error: unknown type name 'time_t'
    time_t          mtime;             /* Time of last modification.        */
    ^
./chansrv_fuse.h:41:5: error: unknown type name 'time_t'
    time_t          ctime;             /* Time of last status change.       */
    ^
3 errors generated.
*** Error code 1

Stop.
make[3]: stopped in /tmp/xrdp/sesman/chansrv
*** Error code 1
```

**Comment by metalefty** [comment]
At least on FreeBSD, it fails to build.

**Comment by metalefty** [comment]
CC: @matt335672
